src/zc/buildout/rmtree.py

```diff
@@ -55,7 +57,18 @@ def rmtree (path):
     """
     def retry_writeable (func, path, exc):
         os.chmod (path, 384) # 0600
-        func (path)
+        tries = 10
+        while tries:
```

Comment:
You can save the use and manipulation of the `tries` variable (i.e. save two lines) by using a `for` loop:

        for i in range(10):
            try:
                func (path)
                break
            except OSError:
                time.sleep(0.1)
        else:
            # tried 10 times without success, thus
            # finally rethrow the last exception
            raise

Reply:
Thx.. commited your suggestion.